server/views/pages/home/index.njk

```diff
@@ -11,7 +11,7 @@
         <div class="govuk-grid-column-full">
             <h1 class="govuk-heading-l">Activities management</h1>
 
-            <div class="card-group">
+            <div class="card-group govuk-grid-row">
```

Comment:
I added this to align cards with layout width on the homepage. Currently there is unneeded padding.

<img width="458" alt="Screenshot at Mar 08 15-31-41" src="https://user-images.githubusercontent.com/125488090/223756421-8185dd81-3ba2-4dac-afa6-733ee4a0b38c.png">